account_asset_management/models/account_move.py

```diff
@@ -81,7 +81,6 @@ def _prepare_asset_vals(self, aml):
             "purchase_value": depreciation_base,
             "partner_id": aml.partner_id,
             "date_start": self.date,
-            "account_analytic_id": aml.analytic_account_id,
```

Comment:
Replace it by the corresponding feature.

Reply:
done